### DISCUSSION TITLE
Question about create-next-app command in README

### DISCUSSION THREAD

**Discussion by 11215548** [chakra-ui/chakra-ui#9376] [Q&A] [answered]
### Description

Hi Chakra UI team,
I'm a user of Chakra UI and I really appreciate the library! I was following your documentation and examples.
The README.md files for both the next-pages example ([https://github.com/chakra-ui/chakra-ui/tree/main/sandbox/next-pages](https://www.google.com/url?sa=E&q=https%3A%2F%2Fgithub.com%2Fchakra-ui%2Fchakra-ui%2Ftree%2Fmain%2Fsandbox%2Fnext-pages)) and the next-app example ([https://github.com/chakra-ui/chakra-ui/tree/main/sandbox/next-app](https://www.google.com/url?sa=E&q=https%3A%2F%2Fgithub.com%2Fchakra-ui%2Fchakra-ui%2Ftree%2Fmain%2Fsandbox%2Fnext-app)) appear to be identical.
The README.md within the next-app example ([https://github.com/chakra-ui/chakra-ui/tree/main/sandbox/next-app](https://www.google.com/url?sa=E&q=https%3A%2F%2Fgithub.com%2Fchakra-ui%2Fchakra-ui%2Ftree%2Fmain%2Fsandbox%2Fnext-app)) suggests using the command:`npx create-next-app --example with-typescript with-typescript-app`

However, when using this command, it actually creates a **Next.js project using the Pages Router**, not the App Router.

### Link to Reproduction

https://stackblitz.com/edit/chakra-ui-v3?file=src%2FApp.tsx 

### Chakra UI Version

^3.2.3

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment, marked as answer]
Those sandbox READMEs were stale. Do not use the `create-next-app --example with-typescript` line.

Follow the current Next App Router guide: https://chakra-ui.com/docs/get-started/frameworks/next-app
